Added a script to delete document from DP&U.

dharmez opened the pull request:
Added a bash script to delete a document from DP&U. Updated README.md.

eyalbenivri commented on sample-deployments/composer-orchestrated-process/DEPLOYMENT.md:
```diff
@@ -1,68 +1,4 @@
-# Deployment Guide
+# [Deployment Guide](../../README.md#deployment-guide)
```
Is there a `#deployment-guide` in the `../../README.md` file? I can't see it

dharmez replied:
Yes, https://github.com/GoogleCloudPlatform/document-processing-and-understanding/blob/delete-doc/README.md#deployment-guide

eyalbenivri reviewed (commented):
Left one comment about an anchor I think might be missing from the README.md file.

dharmez reviewed (commented):
Tested delete doc and restructured some documentation.

anuradha-bajpai-google reviewed (approved):
LGTM

dharmez commented:
* Added a script to delete document from DP&U.

* Removed redundant documentation so that it does not get out of sync.

* Removed some more redundant documentation.

* Update READEME.md.
